spicy/toolchain/src/compiler/codegen/parser-builder.cc

```diff
@@ -726,6 +726,53 @@ struct ProductionVisitor
     // Returns a boolean expression that's 'true' if a 'stop' was encountered.
     Expression parseProduction(const Production& p) { return _parseProduction(p, p.meta()); }
 
+    void skipRegExp(const hilti::ctor::RegExp& c) {
+        auto re = ID("__re");
+        int i = 0;
+        while ( pb->cg()->haveAddedDeclaration(re) )
+            re = ID(fmt("__re_%" PRId64, ++i));
```

Comment:
```suggestion
            re = ID(fmt("__re_%" PRId64, ++i));
            
```

Reply:
What is the change here? These lines seem identical to me.

Reply:
Insert an empty line after the loop body. :-)